torrent/torrent.go

```diff
@@ -156,9 +156,9 @@ func GenerateMetadataInfo(fqfn string) (*MetadataInfo, error) {
 			return nil, err
 		}
 	}
-
 	LogDebug("Generated (or cached) metadata for %s:", fqfn)
 	LogDebug(" * Pieces:     %d * %d bytes", hashCount, PIECE_LENGTH)
+	LogDebug(" * Hashes: %d", len(hashes))
```

Comment:
nit: but you broke the formatting to make things vertically align here (extra spacing).